CMakeLists.txt

```diff
@@ -1,10 +1,72 @@
 # Native code side of Thrive
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.13)
 
-project(Thrive)
+# Must be defined before project() call
+set(CMAKE_CONFIGURATION_TYPES "Debug;Release;RelWithDebInfo;Distribution" CACHE STRING "" FORCE)
 
-# If you want to get compile commands run cmake with
-# "-DCMAKE_EXPORT_COMPILE_COMMANDS=ON"
+project(Thrive VERSION 1.0 LANGUAGES CXX)
```

Comment:
Specifying a version *here* I think is really bad as it will never be updated so it makes no sense to have it here.